flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/OPTIONS/A320_Neo_CDU_OPTIONS_MCDU_KB.js

```diff
@@ -32,6 +32,12 @@ class CDU_OPTIONS_MCDU_KB {
                 mcdu.clearFocus();
                 NXDataStore.set("MCDU_KB_INPUT", "ENABLED");
                 mcdu.addNewMessage(NXFictionalMessages.reloadPlaneApply);
+                const notif = new NXNotif();
+                const popup = new NXPopUp();
+                const showNotif = () => {
+                    notif.showNotification({title: "RELOAD AIRCRAFT", theme: "GAMEPLAY", message:"Reload the aircraft to apply settings"});
+                };
+                popup.showPopUp({ title: "CRITICAL SETTING CHANGED", message: "Please reload the aircraft to apply your new settings.", style: "small"}, showNotif, showNotif);
```

Comment:
**thought:** The exact same code is repeated twice. Can we extract this to one function ?